### PR TITLE
[Docs] Lint Style Guide

### PR DESCRIPTION
**Pull request by alexfornuto**
Corrects instances of `\n` per [markdownlink](https://github.com/DavidAnson/markdownlint/)

### CODE REVIEW

**Review by ptgott — APPROVED** [review]

**Comment by ptgott** [comment]
@alexfornuto Approved, though if we need to be linting for line break style, I suggest we use something that's automated and consistent across all docs PRs, e.g., adding [`remark-lint-linebreak-style`](https://www.npmjs.com/package/remark-lint-linebreak-style/v/3.1.1) to our docs linter. What do you think?

**Comment by alexfornuto** [comment]
@ptgott it's not something we currently "need' as our current docs stack handles fine without proper linebreak style. But I am in favor of automating linting for it and making it consistent. The better formatted our Markdown is, the better prepared we are if/when our SSG stack changes. I just happened to be looking at this file, and wanted to make all the squiggly lines go away 😆

**Comment by alexfornuto** [comment]
This sat for so long waiting for review that it's out of date with recent changes, and not worth manually updating. I can/will take another pass once https://github.com/gravitational/docs/issues/176 is handled.

**Pull request closed without merging**
